recipes/pyscf/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "pyscf" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyscf/pyscf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6a152860787351aae52dc1ef2399bd9564fa314dca070312cfc0f2c3fc7412cf
+  patches:
+    - upstream-osx.patch
+    # https://github.com/pyscf/pyscf/pull/1101
+    - upstream-test.patch
+    # https://github.com/pyscf/pyscf/pull/1142
+    - upstream-linux.patch
+    # https://github.com/pyscf/pyscf/pull/1185
+
+build:
+  number: 0
+  skip: True  # [win or py<39 or osx]
```

Comment:
`py<39` will be removed on the feedstock as the software supports more, the different python builds here just aren't separated enough and the build system pulls in the wrong one.